container/container_contract.go

```diff
@@ -391,17 +419,24 @@ func Owner(containerID []byte) []byte {
 func Count() int {
 	count := 0
 	ctx := storage.GetReadOnlyContext()
-	it := storage.Find(ctx, []byte{}, storage.KeysOnly)
+	it := storage.Find(ctx, []byte(containerKeyPrefix), storage.KeysOnly)
 	for iterator.Next(it) {
-		key := iterator.Value(it).([]byte)
-		// V2 format
-		if len(key) == containerIDSize {
-			count++
-		}
+		count++
 	}
 	return count
 }
 
+// Iterate iterates over all container IDs owned by the specified owner.
+func Iterate(owner []byte) iterator.Iterator {
+	ctx := storage.GetReadOnlyContext()
+
+	if len(owner) == 0 {
+		return storage.Find(ctx, []byte(containerKeyPrefix), storage.KeysOnly|storage.RemovePrefix)
```

Comment:
Wow, this differs a lot from `ValuesOnly` below, this shouldn't be done, we want iterators to have consistent values. If you need some different behavior just add another method, it's not hard to do that. And makes this one only accept correct owners (like `tokensOf` does).

Reply:
I mirrored `List` logic here, but we can iterate over `ownerKeyPrefix` too, fixed.